x-pack/test/security_solution_api_integration/test_suites/detections_response/default_license/rule_execution_logic/execution_logic/machine_learning.ts

```diff
@@ -273,5 +273,25 @@ export default ({ getService }: FtrProviderContext) => {
         expect(fullAlert?.host?.risk?.calculated_score_norm).toBe(1);
       });
     });
+
+    describe('with asset criticality', async () => {
+      before(async () => {
+        await esArchiver.load('x-pack/test/functional/es_archives/asset_criticality');
```

Comment:
If these tests hinge on a specific foreign key across these data sources, it would be good to call that out (similar to how @marshallmain does with [`ID` here](https://github.com/elastic/kibana/blob/main/x-pack/test/security_solution_api_integration/test_suites/detections_response/default_license/rule_execution_logic/execution_logic/query.ts#L72-L76).